### ISSUE TITLE
[Docs] "arg_scope" overrides "defined in" paths

### ISSUE THREAD

**Issue by Androbin** [tensorflow/tensorflow#10398]
I found that some ops have incorrect "defined in" paths in the [`contrib.layers` docs](https://www.tensorflow.org/api_docs/python/tf/contrib/layers).

Examples: `avg_pool2d, batch_norm, bias_add, conv2d, conv2d_in_plane, conv2d_transpose, dropout, flatten, fully_connected, layer_norm, one_hot_encoding, separable_conv2d, softmax`

It always uses [`tensorflow/contrib/framework/python/ops/arg_scope.py`](https://github.com/tensorflow/tensorflow/blob/r1.1/tensorflow/contrib/framework/python/ops/arg_scope.py) which is clearly outright wrong.

This seems to be the case for all ops defined in [`tensorflow/contrib/layers/python/layers/layers.py`](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/layers/python/layers/layers.py) which are annotated with `@add_arg_scope`

**Comment by jart** [comment]
@dr4b Our friend has identified has at least thirteen functions in contrib where the "defined in" link needs to be updated.

**Comment by dr4b** [comment]
I don't own the tool that generates these; reassigning to Mark to take a look.

**Comment by Androbin** [comment]
Seems to be resolved now, closing.
